### DISCUSSION TITLE
ImportError

### DISCUSSION THREAD

**Discussion by andalbrue** [jkctech/Telegram-Trilateration#3] [General]
I have tried to execute the file  "python scraper.py" but show an error in prompt:
     "ImportError: cannot import name 'settings' from 'utils'"

Can you tell me whats is the problem or How i can to do to resolve it

(I have already installed "pip install utils")

Thanks

**Comment by jkctech** [comment]
"utils" refers to the folder "utils".

"settings" refers to the settings.py file inside that folder.

If you uninstall "utils" it should work.

Sidenote: I will post a much eadier version to work with today, maybe you prefer that since the scraper is not really recommended to be used 😅

**Reply by jkctech** [reply]
https://github.com/jkctech/Telegram-Trilateration/tree/master/API :)
